src/settings-ui/Settings.UI/OOBE/Views/OobeWhatsNew.xaml.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.PowerToys.Settings.UI.Library.Utilities;
+using Microsoft.PowerToys.Settings.UI.OOBE.Enums;
+using Microsoft.PowerToys.Settings.UI.OOBE.ViewModel;
+using Microsoft.PowerToys.Settings.UI.Views;
+using Windows.ApplicationModel.Resources;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace Microsoft.PowerToys.Settings.UI.OOBE.Views
+{
+    public sealed partial class OobeWhatsNew : Page
+    {
+        // Contains information for a release. Used to deserialize release JSON info from GitHub.
+        private class PowerToysReleaseInfo
+        {
+            [JsonPropertyName("published_at")]
+            public DateTimeOffset PublishedDate { get; set; }
+
+            [JsonPropertyName("name")]
+            public string Name { get; set; }
+
+            [JsonPropertyName("tag_name")]
+            public string TagName { get; set; }
+
+            [JsonPropertyName("body")]
+            public string ReleaseNotes { get; set; }
+        }
+
+        public OobePowerToysModule ViewModel { get; set; }
+
+        public OobeWhatsNew()
+        {
+            this.InitializeComponent();
+            ViewModel = new OobePowerToysModule(OobeShellPage.OobeShellHandler.Modules[(int)PowerToysModulesEnum.WhatsNew]);
+            DataContext = ViewModel;
+        }
+
+        private async Task<string> GetReleaseNotesMarkdown()
+        {
+            string releaseNotesJSON = string.Empty;
+            using (HttpClient getReleaseInfoClient = new HttpClient())
+            {
+                // GitHub APIs require sending an user agent
+                // https://docs.github.com/en/rest/overview/resources-in-the-rest-api#user-agent-required
+                getReleaseInfoClient.DefaultRequestHeaders.TryAddWithoutValidation("User-Agent", "PowerToys");
+                releaseNotesJSON = await getReleaseInfoClient.GetStringAsync("https://api.github.com/repos/microsoft/PowerToys/releases");
+            }
+
+            IList<PowerToysReleaseInfo> releases = JsonSerializer.Deserialize<IList<PowerToysReleaseInfo>>(releaseNotesJSON);
+
+            // Get the latest releases
+            var latestReleases = releases.OrderByDescending(release => release.PublishedDate).Take(5);
```

Comment:
nit: Not sure if 5 is to much.. Maybe 3?

Reply:
Idea here is that people sometimes wait some versions to upgrade and we do some hotfixes. (This last one was two hotfixes). 5 felt like a good compromise to make sure people are able to see 2 releases worth of features.